pkg/process/v5/writer.go

```diff
@@ -13,22 +15,36 @@ import (
 	"github.com/anchore/grype-db/internal/file"
 	"github.com/anchore/grype-db/internal/log"
 	"github.com/anchore/grype-db/pkg/data"
+	"github.com/anchore/grype-db/pkg/provider"
 	"github.com/anchore/grype/grype/db"
 	grypeDB "github.com/anchore/grype/grype/db/v5"
 	grypeDBStore "github.com/anchore/grype/grype/db/v5/store"
 )
 
 // TODO: add NVDNamespace const to grype.db package?
-const nvdNamespace = "nvd:cpe"
+const (
+	nvdNamespace             = "nvd:cpe"
+	ProviderMetadataFileName = "provider-metadata.json"
```

Comment:
this should be unexported 
```suggestion
	providerMetadataFileName = "provider-metadata.json"
```